core/Pos/Core/Address.hs

```diff
@@ -372,3 +378,25 @@ isBootstrapEraDistrAddress (addrAttributesUnwrapped -> AddrAttributes {..}) =
     case aaStakeDistribution of
         BootstrapEraDistr -> True
         _                 -> False
+
+----------------------------------------------------------------------------
+-- Maximal size
+----------------------------------------------------------------------------
+
+-- | Maximal size of PubKey address with BootstrapEra
+-- distribution. Actual size depends on CRC32 value which is
+-- serialized using var-length encoding.
+maxPubKeyAddressSizeBoot :: Byte
+maxPubKeyAddressSizeBoot = 43
```

Comment:
How can one ensure these values make sense? Can you maybe add a short notice on how they were calculated, so anybody can validate numbers without applying too much effort?

Reply:
Hm, I think I should add tests for these values where addresses are generated from fixed values and their lengths are asserted.